content/en/container.md

```diff
@@ -12,10 +12,7 @@ but typically the operating system prevents the separate container processes fro
 
 ## Problem it addresses
 
-Before containers were available, separate machines were necessary to run applications. 
-Each machine would require its own operating system, which takes CPU, memory, and disk space, 
-all for an individual application to function. 
-Additionally, the maintenance, upgrade, and startup of an operating system is another significant source of toil. 
+Containers enable consistent deployment and execution across diverse machines. By abstracting the user space and not the entire operating system, containers eliminate the toil of starting and maintaining independent instances of operating systems, enabling the next computing model after virtualization. 
```

Comment:
In the first sentence you are talking about a solution, where we aim to focus on the problem in this section. Also you are introducing the user space which is a term that is not necessary to understand containers and may confuse especially non-tech people or beginners. We aim for simple and understandable language. Speaking of which, the second sentence is super long and would benefit from being split in multiple sentences. The last part of the second sentence is correct and does not describe any problem that containers try to solve. It could be added to the how it helps section maybe. Last but not least, please make sure to use semantic line breaks when you make contributions :) Thanks